lib/between_meals/repo/hg.rb

```diff
@@ -115,9 +115,17 @@ def head_parents
       end
 
       def last_author
-        { :email => template('author').match(/^(.*)<(.*)>$/)[2] }
-      rescue
-        { :email => nil }
+        # rubocop:disable Lint/HandleExceptions
+        begin
+          return { :email => template('author').match(/^(.*)<(.*)>$/)[2] }
+        rescue
```

Comment:
why not rescue only the Match exception here - you know exactly what you're looking for - and then you wouldn't need the rubocop exceptions